src/components/sections/PartnerLogo.tsx

```diff
@@ -1,87 +1,6 @@
 import React from "react";
```

Comment:
This is the file that we'll be referencing in the future so can we leave this untouched?